examples/tensorflow/common/models.py

```diff
@@ -61,7 +61,7 @@ def MobileNetV3(stack_fn, last_point_ch, input_shape=None, model_type='large', *
     se_ratio = 0.25
 
     x = img_input
-    x = layers.Rescaling(scale=1. / 127.5, offset=-1.)(x)
+    x = layers.Rescaling(1. / 255.)(x)
```

Comment:
These changes will break accuracy for mobilenet_v3. Please, revert these changes.

Reply:
Reverted.